monai/apps/datasets.py

```diff
@@ -241,11 +245,28 @@ def __init__(
                 f"Cannot find dataset directory: {dataset_dir}, please use download=True to download it."
             )
         data = self._generate_data_list(dataset_dir)
+        if property_keys is not None:
+            self._properties = load_decathlon_properties(os.path.join(dataset_dir, "dataset.json"), property_keys)
```

Comment:
I think we should always load all the properties, this is only a very small amount of data and keep the function signature simple

Reply:
OK, updated in the latest commit.
Thanks.